### DISCUSSION TITLE
using --env-file doesn't seem to work but -v does???

### DISCUSSION THREAD

**Discussion by gmulligan** [dani-garcia/vaultwarden#1507] [Q&A]
if I try to use podman (or docker)  run with --env-file /home/bitwardenrs/config it complains that there is no .env file and I can't connect to the bitwarden server.

if I use podman (or docker) run with -v /home/bitwardenrs/config:/.env everything works just great.

**Comment by BlackDex** [comment]
Could you give us a bit more information. Some logs right after starting etc..
What happens if you start it with a minimal env-file?
So something like:
```bash
LOG_LEVEL=debug
DISABLE_ADMIN_TOKEN=true
```

That should at least start the container with debug log-level and you should be able to access the /admin interface (via IP or something) without a token.
